### DISCUSSION TITLE
Feature request: Force controller to be plugged in

### DISCUSSION THREAD

**Discussion by MastaG** [darthcloud/BlueRetro#1053] [Ideas]
I'm using a BlueRetro on my Sega Dreamcast, running the latest git version which allows for Rumble and VMU emulation.
It would be great if there I could thick a box somewhere which force the first controller (P1) to always be connected, regardless whether the bluetooth controller is paired or not.
Reason for this is that the VMU will be unavailable if the Dreamcast doesn't have a controller plugged in.
Also some KOS applications such as DreamShell don't support hotplugging with the BlueRetro, the controller should be present before it starts or else it doesn't work.

**Comment by darthcloud** [comment]
Yes I should add so config for turning hotplug off. I made a bug to track this: #1168
